src/main/java/uk/gov/justice/hmpps/casenotes/dto/CaseNote.java

```diff
@@ -76,8 +76,76 @@
     @ApiModelProperty(value = "Location Id representing where Case Note was made.", position = 14, example = "MDI")
     private String locationId;
 
+    @ApiModelProperty(required = true, value = "Delius number representation of the case note id - will be negative for sensitive case note types", position = 15, example = "-23")
+    @NotNull
+    private Integer eventId;
+
     @ApiModelProperty(required = true, value = "Ordered list of amendments to the case note (oldest first)", position = 15)
     @NotNull
     @Builder.Default
     private List<CaseNoteAmendment> amendments = new ArrayList<>();
+
+    public @NotNull String getCaseNoteId() {
```

Comment:
Unfortunately lombok processor happens after kotlin processor so kotlin can't access getters, so have to delombok.  Will be fixed when we move to kotlin data class instead...